### ISSUE TITLE
False positive regarding center/centre in LaTeX document

### ISSUE THREAD

**Issue by mtb-za** [amperser/proselint#533]
I am using LaTeX, but with my habitual South African spelling (which is essentially en-uk). Proselint is picking up my use of "centre" as being inconsistent, I think it may be because I am using things like the following:

    \begin{table}[hbt!]
    \begin{center}
    \caption{Structural Index (SI) values for ideal geometries.}
    \label{tab:SI_values}
    \begin{tabular}{lll}
        Ideal Geometry Model & Magnetic SI & Gravity SI\\
        \hline
        Point, sphere & 3 & 2 \\
        Line, cylinder, thin-bed fault & 2 & 1 \\
        Thin sheet edge, thin sill, thin dyke & 1 & 0
    \end{tabular}
    \end{center}
    \end{table}

This is definitely a false positive, since nowhere in my text do I use "center", only in the LaTeX command.

**Comment by suchow** [comment]
Yes, this is a bug — Proselint does not yet ignore markup from LaTeX files. Soon I'll aggregate similar issues that have been submitted, and with any luck we'll get this fixed. 

Thanks.

**Comment by Decagon** [comment]
For a quick fix, you could paste the following code into a separate latex file called `centre.tex`:

```
\newcommand{\begincentre}{\begin{center}}
\newcommand{\endcentre}{\end{center}}
```

Then in your latex file, call `\input{"centre.tex"}` after `\begin{document}`.

**Comment by languitar** [comment]
Are there any suppression inline comments that can be used to temporarily ignore such issues?
